lib/whois/parsers/whois.isoc.org.il.rb

```diff
@@ -60,7 +60,16 @@ class WhoisIsocOrgIl < Base
         end
       end
 
-      property_not_supported :expires_on
+      property_supported :expires_on do
+        if content_for_scanner =~ /validity:\s+(.+)\n/
+          case $1.downcase
+          when "N/A"
```

Comment:
Can you provide a fixture where there is a domain with N/A? I want to make sure this is still in use and it's properly tested.